diff --git a/common/drivers/Kconfig b/common/drivers/Kconfig
index 0d399ddaa185..a4c76130a4ee 100644
--- a/common/drivers/Kconfig
+++ b/common/drivers/Kconfig
@@ -236,4 +236,6 @@ source "drivers/interconnect/Kconfig"
 source "drivers/counter/Kconfig"
 
 source "drivers/most/Kconfig"
+
+source "drivers/terminalsu/Kconfig"
 endmenu
diff --git a/common/drivers/Makefile b/common/drivers/Makefile
index a110338c860c..d21697addbd0 100644
--- a/common/drivers/Makefile
+++ b/common/drivers/Makefile
@@ -187,3 +187,4 @@ obj-$(CONFIG_GNSS)		+= gnss/
 obj-$(CONFIG_INTERCONNECT)	+= interconnect/
 obj-$(CONFIG_COUNTER)		+= counter/
 obj-$(CONFIG_MOST)		+= most/
+obj-$(CONFIG_TSU) += terminalsu/
